pretext/UnitTesting/Exercises.ptx

```diff
@@ -67,9 +67,6 @@ myTests().main()
             <p>If you're not sure how to get started with this one, review <xref ref="unit-testing_writing-unit-tests"/>.</p>
           </hint>
           <solution>
-                        assert reverse('abc') == 'cba'
-assert reverse('b') == 'b'
-assert reverse('') == ''
```

Comment:
these are good tests?  I think you should leave them in unless I'm not understanding something.

Reply:
They are repeated, there is another copy of the tests still there. I only deleted the extra copy of it so two do not appear in the solution window.

Reply:
OK, I didn't get that from looking at the diff.

Thanks for clarifying.